### PR TITLE
support firmware rpm filter name changes, minor cleanups

### PR DESCRIPTION
**Pull request by ctzsnooze**
Changes:

- Supports the rpm filteringname changes from [10856](https://github.com/betaflight/betaflight/pull/10856).
- Includes `rpm_filter_fade_range_hz` parameter
- Tidies up the gyro lowpass filtering html
- Moves DShot min and max values to the Motors section, leaving idle values in the Throttle section.
- Updates comments in `flightlog_parser.js` about adding new parameters
- Whitespace tidy ups

Image showing `rpm_filter_fade_range` element:

Screen Shot 2021-09-17 at 17 31 57

Image showing the motor and throttle rearrangement
Screen Shot 2021-09-17 at 17 22 23
:

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: js/header_dialog.js
Comment (haslinghuis) [inline js/header_dialog.js:634]: comma space please

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: js/header_dialog.js
Comment (ctzsnooze) [reply js/header_dialog.js:634]: it's 'hit and miss' throughout the document, most lines are `,0);` and some are ` 0);`.
I can make them consistent but it will change hundreds of lines :-)

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
Closing since these changes were included in https://github.com/betaflight/blackbox-log-viewer/pull/539, merged recently

**Pull request closed without merging**
